homeassistant/components/systemmonitor/coordinator.py

```diff
@@ -84,15 +84,28 @@ def __init__(
         )
         self._psutil = psutil_wrapper.psutil
         self._arguments = arguments
+        self.boot_time: datetime | None = None
+
+        self._initial_update: bool = True
+        self.update_list: list[tuple[str, str]] = []
```

Comment:
Maybe make this a dict with each subscribed entity being the keys

`self.update_subscribers = dict[tuple[str,str], set[str]]`

The `set` could be the subscriber unique ids

than you can do `("cpu_percent", "") in self.update_subscribers` and it would not have to linearly search the list